src/main/java/space/obminyashka/items_exchange/service/impl/AdvertisementServiceImpl.java

```diff
@@ -63,6 +65,20 @@ public List<AdvertisementTitleDto> findRandom12Thumbnails() {
         return findAllThumbnails(PageRequest.of(randomPage, resultsQuantity)).getContent();
     }
 
+    @Override
+    @Cacheable
```

Comment:
We have a task for removing caching, don't apply it just because it was done for another endpoint.